packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -666,8 +666,11 @@ class StackViewport extends Viewport {
   private calibrateIfNecessary(imageId, imagePlaneModule) {
     const calibration = metaData.get('calibratedPixelSpacing', imageId);
     const isUpdated = this.calibration !== calibration;
-    const { scale } = calibration || {};
-    this.hasPixelSpacing = scale > 0 || imagePlaneModule.rowPixelSpacing > 0;
+    const scale = calibration?.scale;
+    this.hasPixelSpacing =
+      scale > 0 ||
+      (!imagePlaneModule.usingDefaultValues &&
```

Comment:
https://github.com/OHIF/Viewers/blob/afa9131da1d6fccaf5f72c66dcf3d83b2b0e5bfa/platform/core/src/classes/MetadataProvider.ts#L549

Are we still sure we want to trust `usingDefaultValues` to drive this? It seems to be set to true if any of several conditions pass here in the OHIF metadata provider.

If we will, can we make sure if that `usingDefaultValues` is `true`, then there's geniunely no pixel spacing, and it wasn't set to true due to some other missing field.